docs/installation/docker.md

```diff
@@ -55,8 +55,8 @@ well.
      `config/application.yml`:
 
      ```shell
-      REDIS_URL: "redis://redis:6379"
-      REDIS_SESSIONS_URL: "redis://redis:6379"
+      REDIS_URL: "redis://localhost:6379"
```

Comment:
I haven't been able to successfully run DEV in docker yet. I have problems pulling down the `alpine-ruby-node` image so I'm not sure if `redis` is mapped to `localhost` in the Docker container.

Reply:
This shouldn't be changed, `redis` is a virtual host created by `docker-componse.yml`, see this issue https://github.com/thepracticaldev/dev.to/issues/4962 which lead to https://github.com/thepracticaldev/dev.to/pull/4965

How are you starting the Docker app?

Reply:
I'm running `./docker-run.sh DEV`.

It seems to have issues pulling an image down.

```
#---
#
# This script will perform the following steps ... 
#
# 1) Stop and remove any docker container with the name 'dev-to-postgres' and 'dev-to'
# 2) Reset any storage directories if RUN_MODE starts with 'RESET-'
# 3) Build the dev.to docker image, with the name of 'dev-to:dev' or 'dev-to:demo'
# 4) Deploy the postgres container, mounting '_docker-storage/postgres' with the name 'dev-to-postgres'
# 5) Deploy the dev-to container, with the name of 'dev-to-app', and sets up its port to 3000
#
# To run this script properly, execute with the following (inside the dev.to repository folder)...
# './docker-run.sh [RUN_MODE] [Additional docker environment arguments]'
#
# Alternatively to run this script in 'interactive mode' simply run
# './docker-run.sh INTERACTIVE-DEMO'
#
#---
#---
#
# RUN_MODE can either be the following
#
# - 'DEV'  : Start up the container into bash, with a quick start guide
# - 'DEMO' : Start up the container, and run dev.to (requires ALGOLIA environment variables)
# - 'RESET-DEV'   : Resets postgresql and upload data directory for a clean deployment, before running as 
DEV mode
# - 'RESET-DEMO'  : Resets postgresql and upload data directory for a clean deployment, before running as 
DEMO mode
# - 'INTERACTIVE-DEMO' : Runs this script in 'interactive' mode to setup the 'DEMO'
#
# So for example to run a development container in bash it's simply
# './docker-run.sh DEV'
#
# To run a simple demo, with some dummy data (replace <?> with the actual keys)
# './docker-run.sh DEMO -e ALGOLIASEARCH_APPLICATION_ID=<?> -e ALGOLIASEARCH_SEARCH_ONLY_KEY=<?> -e ALGOLI
ASEARCH_API_KEY=<?>'
#
# Finally to run a working demo, you will need to provide either...
# './docker-run.sh .... -e GITHUB_KEY=<?> -e GITHUB_SECRET=<?> -e GITHUB_TOKEN=<?>
#
# And / Or ...
# './docker-run.sh .... -e TWITTER_ACCESS_TOKEN=<?> -e TWITTER_ACCESS_TOKEN_SECRET=<?> -e TWITTER_KEY=<?> 
-e TWITTER_SECRET=<?>
#
# Note that all of this can also be configured via ENVIRONMENT variables prior to running the script
#
#---
#---
# Ok, to start with - lets assume the following settings (provided or auto default)...
#
# RUN_MODE     = DEV
# REPO_DIR     = /Users/nickytonline/dev/dev.to
# STORAGE_DIR  = /Users/nickytonline/dev/dev.to/_docker-storage
# UPLOAD_DIR   = /Users/nickytonline/dev/dev.to/_docker-storage/public-upload/
# POSTGRES_DIR = /Users/nickytonline/dev/dev.to/_docker-storage/postgresql-data/
#
# PS  : These settings can also be overwritten by ENVIRONMENT variables, extremely useful in a CI setup =)
#---
#---
# Lets scan for dev.to environment variables that will automatically be passed
# forward into the container if present (very useful for CI testing)
#---
[skipped env variable]  - ALGOLIASEARCH_APPLICATION_ID
[skipped env variable]  - ALGOLIASEARCH_SEARCH_ONLY_KEY
[skipped env variable]  - ALGOLIASEARCH_API_KEY
[skipped env variable]  - GITHUB_KEY
[skipped env variable]  - GITHUB_SECRET
[skipped env variable]  - GITHUB_TOKEN
[skipped env variable]  - TWITTER_ACCESS_TOKEN
[skipped env variable]  - TWITTER_ACCESS_TOKEN_SECRET
[skipped env variable]  - TWITTER_KEY
[skipped env variable]  - TWITTER_SECRET
[skipped env variable]  - PUSHER_APP_ID
[skipped env variable]  - PUSHER_KEY
[skipped env variable]  - PUSHER_SECRET
[skipped env variable]  - PUSHER_CLUSTER
#---
# Building dev-to DEV docker container (yay!) ... 
# If this is your first time running this build step, go grab a coffee - it may take a long while
# Alternatively, some paper swords with chairs : https://xkcd.com/303/
#---
Sending build context to Docker daemon  18.27MB
Error response from daemon: failed to reach build target alpine-ruby-node in Dockerfile
```

Reply:
What happens if you use docker-compose? https://docs.dev.to/installation/docker/#running-the-docker-app-(basic)

Reply:
I’ll give it a go later tonight and be basic. 😉

Reply:
> What happens if you use docker-compose? [docs.dev.to/installation/docker/#running-the-docker-app-(basic)](https://docs.dev.to/installation/docker/#running-the-docker-app-(basic))

All is good when running the four `docker-compose` commands.